src/util/auth.js

```diff
@@ -0,0 +1,38 @@
+const AUTH_SESSION_KEY = 'likecoin_nft_book_press_token';
+
+export function loadAuthSession() {
+  try {
+    if (window.localStorage) {
+      const data = window.localStorage.getItem(AUTH_SESSION_KEY);
+      if (data) {
+        const { wallet, token } = JSON.parse(data);
+        return {
+          wallet,
+          token,
+        };
+      }
+    }
+  } catch {}
+
+  return null;
+}
+
+export function saveAuthSession(session) {
```

Comment:
Maybe we should explicitly parse `{ wallet, token }` out, since we only use them in line 8

Reply:
[♻️ Update saveAuthSession to only store wallet and token](https://github.com/likecoin/liker-land/pull/1823/commits/74f2e72f79203cc858ddf53d03d44e75d11ed51a)